rfcs/0021-ckb-address-format/0021-ckb-address-format.md

```diff
@@ -38,13 +38,28 @@ payload = 0x01 | code_hash_index | single_arg
 To translate payload to lock script, one can convert code_hash_index to code_hash and hash_type with the following *popular code_hash table*. And single_arg as the args.
 
 
-| code_hash_index |        code_hash     |   code_type  |      args     |
-|:---------------:|----------------------|:------------:|---------------|
-|      0x00       | SECP256K1 + blake160 |     Type     |  blake160(PK) |
-|      0x01       | SECP256K1 + hash160  |     Type     |  hash160(PK)  |
+| code_hash_index |        code_hash     |   code_type  |          args           |
+|:---------------:|----------------------|:------------:|-------------------------|
+|      0x00       | SECP256K1 + blake160 |     Type     |  blake160(PK)*          |
+|      0x01       | SECP256K1 + multisig |     Type     |  multisig script hash** |
 
 \* The blake160 here means the leading 20 bytes truncation of Blake2b hash result.
-\* The hash160 here means Bitcoin address calculation *double hash* algorithm, which means `hash160(PK) := Ripemd160(SHA256(PK))`.
+
+\*\* The *multisig script hash* is the 20 bytes blake160 hash of multisig script. The multisig script should be assembled in the following format:
+
+```
+S | R | M | N | Pubkey1 | Pubkey2 | ...
```

Comment:
Please note the multisig script is changed to use the blake160 hash of public keys here instead of directly using public keys: https://github.com/nervosnetwork/ckb-system-scripts/pull/61

Reply:
I will modify the expression.